.github/workflows/ci-build.yml

```
@@ -63,4 +63,18 @@ jobs:
         skip-build-cache: true
         args: --config=./.golangci.yml --verbose
 
+  gomodreplacements:
+    name: go.mod replacements
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    
+    - name: check
+      run: |
+        if [[ -n "$(grep 'replace github.com/codeready-toolchain/.*' go.mod || true)" ]]; then
+          echo "forbidden replacement in go.mod"
```

shouldn't we also exit with a non-zero value?
```suggestion
          echo "forbidden replacement in go.mod"
          exit 1
```